### DISCUSSION TITLE
Linkvertise isn't skipping

### DISCUSSION THREAD

**Discussion by Ash-627** [FastForwardTeam/FastForward#1381] [Q&A] [answered]
So, I have just downloaded FastForward to skip some linkvertise stuff, however, it is not skipping anything. I activated all recommended options and made the time 0 seconds, 1 second, 2 seconds, and nothing was skipped.
![image](https://github.com/FastForwardTeam/FastForward/assets/113243939/cf323c9b-7fd5-49d5-9ec0-0a63d686d1d3)
So, how can I do this now?

Any help would be appreciated.

**Comment by Rust1667** [comment, marked as answer]
read the https://github.com/FastForwardTeam/FastForward/blob/main/README.md
use the first alternative mentioned
